content/04.results.md

```diff
@@ -2,42 +2,54 @@
-The CtD edge type is a quintessential example of this trend.
-The baseline model starts off with an AUROC score of 52% and an AUPRC of 28%, which increase to 76% and 49% respectively as more CtD label functions are included. 
-DaG edges have a similar trend: performance starting off with an AUROC of 56% and AUPR of 41% then increases to 62% and 45% respectively.
-Both the CbG and GiG edges have an increasing trend but plateau after a few label functions are added.  
-
-The off-diagonals in figure {@fig:auroc_gen_model_performance} and supplemental figure {@fig:aupr_gen_model_performance} show how performance varies when label functions from one edge type are added to a different edge type's baseline.
-In certain cases (apparent for DaG), performance increases regardless of the edge type used for label functions.
-In other cases (apparent with CtD), one label function appears to improve performance; however, adding more label functions does not improve performance (AUROC) or decreases it (AUPR).
-In certain cases, the source of the label functions appears to be important: the performance of CbG edges decrease when using label functions from the DaG and CtD categories.
-
-Our initial hypothesis was based on the idea that certain edge types capture similar physical relationships and that these cases would be particularly amenable for label function transfer.
-For example, CbG and GiG both describe physical interactions.
-We observed that performance increased as assessed by both AUROC and AUPR when using label functions from the GiG edge type to predict CbG edges.
-A similar trend was observed when predicting the GiG edge; however, the performance differences were small for this edge type making the importance difficult to assess.  
-The last column shows increasing performance (AUROC and AUPR) for both DaG and CtD when sampling from all label functions.
-CbG and GiG also had increased performance when one random label function was sampled, but performance decreased drastically as more label functions were added.
-It is possible that a small number of irrelevant label functions are able to overwhelm the distant supervision label functions in these cases (see Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
-
-### Random Label Function Generative Model Analysis
+We reported the results in AUROC and AUPR (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}; Supplemental Figures {@fig:auroc_gen_model_tune_set} and {@fig:auroc_gen_model_tune_set}).  
+Performance increases as the number of edge-specific label functions is added to the baseline model (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}). 
```

Comment:
```suggestion
Performance increases when edge-specific label functions are added to an edge-specific baseline model trained with distant supervision (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}). 
```
You could probably move either AUPR or AUROC to the supplement. I'm not sure what having both figures here is adding, unless there's a difference.